### PR TITLE
[HOTFIX] Upgraded jars to work S3 with presto

### PR DESCRIPTION
**Pull request by ravipesala**
There is a duplicate jar `aws-java-sdk` and low version jars avoid connecting to S3 in presto. Those jars are upgraded in this PR and updated doc.

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2717/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2488/

File: core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java [hunk 1]
```diff
@@ -655,7 +655,7 @@ private long getEndTimeOfDeltaFile(String extension, SegmentUpdateDetails block)
     // get the updated status file identifier from the table status.
     String tableUpdateStatusIdentifier = getUpdatedStatusIdentifier();
 
-    if (null == tableUpdateStatusIdentifier) {
+    if (null == tableUpdateStatusIdentifier || tableUpdateStatusIdentifier.length() == 0) {
```
Comment (qiuchenjian) [inline core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java]: StringUtils.isNotEmpty is better

File: docs/presto-guide.md [hunk 2]
```diff
@@ -254,23 +254,15 @@ Now you can use the Presto CLI on the coordinator to query data sources in the c
    ```
     Required properties
 
-    fs.s3a.access.key={value}
-    fs.s3a.secret.key={value}
+    hive.s3.aws-access-key={value}
```
Comment (qiuchenjian) [inline docs/presto-guide.md]: hive.s3.aws is OK ?  if we work on huawei's OBS

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2500/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10758/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2729/

**Comment by kumarvishal09** [comment]
LGTM

**Pull request closed without merging**